test-git.t fails

```
============================= test session starts ==============================
platform linux2 -- Python 2.7.14, pytest-3.4.1, py-1.5.2, pluggy-0.6.0
rootdir: /build/source, inifile: pytest.ini
plugins: flake8-1.0.0, cram-0.2.0
collected 9 items

tests/test-diff.t .                                                      [ 11%]
tests/test-git.t F                                                       [ 22%]
tests/test-hg.t .                                                        [ 33%]
tests/test-keep-count.t .                                                [ 44%]
tests/test-keep-output-keep-count.t .                                    [ 55%]
tests/test-keep-output.t .                                               [ 66%]
tests/test-metadata.t .                                                  [ 77%]
tests/test-unicode.t .                                                   [ 88%]
tests/test-widgets.t .                                                   [100%]

=================================== FAILURES ===================================
___________________________ [cram] tests/test-git.t ____________________________
--- test-git.t
+++ test-git.t.err
@@ -7,7 +7,6 @@
   $ ${NBSTRIPOUT_EXE:-nbstripout} --install
   $ ${NBSTRIPOUT_EXE:-nbstripout} --is-installed
   $ git diff --no-ext-diff --unified=0 --exit-code -a --no-prefix ${TESTDIR}/test_diff.ipynb ${TESTDIR}/test_diff_output.ipynb
-  [1]
   $ git diff ${TESTDIR}/test_diff.ipynb ${TESTDIR}/test_diff_different.ipynb
   (diff --git.*) (re)
   (index .*) (re)

====================== 1 failed, 8 passed in 7.12 seconds ======================
```

The output of `pip freeze`:
```
asn1crypto==0.24.0
attrs==17.4.0
backports.shutil-get-terminal-size==1.0.0
certifi==2018.1.18
cffi==1.11.5
chardet==3.0.4
configparser==3.5.0
cram==0.7
cryptography==2.1.4
decorator==4.2.1
docopt==0.6.2
docutils==0.14
enum34==1.1.6
flake8==3.5.0
funcsigs==1.0.2
functools32==3.2.3.post2
idna==2.6
ipaddress==1.0.18
ipython==5.5.0
ipython-genutils==0.2.0
jsonschema==2.6.0
jupyter-core==4.4.0
mccabe==0.6.1
nbformat==4.4.0
nbstripout==0.3.1
packaging==16.8
path.py==11.0
pathlib2==2.3.0
pexpect==4.4.0
pickleshare==0.7.4
pluggy==0.6.0
prompt-toolkit==1.0.15
ptyprocess==0.5.2
py==1.5.2
pyasn1==0.4.2
pycodestyle==2.3.1
pycparser==2.14
pyflakes==1.6.0
Pygments==2.2.0
pyOpenSSL==17.5.0
pyparsing==2.2.0
PySocks==1.6.6
pytest==3.4.1
pytest-cram==0.2.0
pytest-flake8==1.0.0
pytest-runner==4.0
requests==2.18.4
scandir==1.7
simplegeneric==0.8.1
six==1.11.0
testpath==0.3
traitlets==4.3.2
urllib3==1.22
wcwidth==0.1.6
```

--- a/tests/test-git.t
+++ b/tests/test-git.t
@@ -6,9 +6,8 @@
   [1]
   $ ${NBSTRIPOUT_EXE:-nbstripout} --install
   $ ${NBSTRIPOUT_EXE:-nbstripout} --is-installed
-  $ git diff --no-ext-diff --unified=0 --exit-code -a --no-prefix ${TESTDIR}/test_diff.ipynb ${TESTDIR}/test_diff_output.ipynb 
-  [1]
-  $ git diff ${TESTDIR}/test_diff.ipynb ${TESTDIR}/test_diff_different.ipynb 
+  $ git diff --no-index --no-ext-diff --unified=0 --exit-code -a --no-prefix ${TESTDIR}/test_diff.ipynb ${TESTDIR}/test_diff_output.ipynb
+  $ git diff --no-index ${TESTDIR}/test_diff.ipynb ${TESTDIR}/test_diff_different.ipynb
   (diff --git.*) (re)
   (index .*) (re)
   (--- .*test_diff.ipynb) (re)

